ros2_driver:
  ros__parameters:
    niryo_ros2_interfaces_package: "niryo_ned_ros2_interfaces"
    rosbridge_port: 9090
    robot_namespaces:
      - "robot1"
      - "robot2"
    robot_ips:
      - "192.168.1.140"
      - "192.168.1.189"